core/controllers/suggestion.py

```diff
@@ -40,21 +40,108 @@
 class SuggestionHandler(base.BaseHandler):
     """"Handles operations relating to suggestions."""
 
+    POST_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'suggestion_type': {
+                'schema': {
+                    'type': 'basestring',
+                    'choices': feconf.SUGGESTION_TYPE_CHOICES
+                }
+            },
+            'target_type': {
+                'schema': {
+                    'type': 'basestring',
+                    'choices': feconf.SUGGESTION_TARGET_TYPE_CHOICES
+                }
+            },
+            'target_id': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            },
+            'target_version_at_submission':
+            {
+                'schema': {
+                    'type': 'int'
+                }
+            },
+            'change': {
+               'schema': {
+                   'type': 'dict',
+                   'properties':
+                    [
+                       {
+                        'name': 'cmd',
+                        'schema': {
+                            'type': 'basestring'
+                        }
+                       },
+                       {
+                        'name': 'state_name',
+                        'schema': {
+                            'type': 'basestring'
+                        }
+                       },
+                       {
+                        'name': 'content_id',
+                        'schema': {
+                            'type': 'basestring'
+                        }
+                       },
+                       {
+                        'name': 'language_code',
+                         'schema': {
+                             'type': 'basestring'
+                         },
+                         'default': None
+                       },
+                       {
+                        'name': 'content_html',
+                        'schema': {
+                            'type': 'html'
+                        }
+                       },
+                       {
+                        'name': 'translation_html',
+                        'schema': {
+                            'type': 'html'
+                        }
+                       },
+                       {
+                        'name': 'data_format',
+                        'schema': {
+                            'type': 'basestring'
+                        }
+                       }
```

Comment:
Fix the formatting here, as I suggested in your previous PR.

Reply:
Done.